guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -137,7 +137,7 @@ endif::[]
 +
 [options="nowrap"]
 ----
-# rpm -qa --last | grep kernel
+# rpm -q --last kernel
```

Comment:
Should we also use the long version for `-q`
```suggestion
# rpm --query --last kernel
```

Reply:
Thank you for your suggestion. In my opinion, however, we should keep the `-q` option to make the command shorter.

Reply:
While in daily life I always use short options, in documentation I always try to use long options because they're self documenting. `-q` is also often used for `--quiet` and with a long option there's no confusion what it does.

Reply:
Noted. I will update the command with the long option.